src/Microsoft.TestPlatform.PlatformAbstractions/netcore/System/PlatformThread.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.PlatformAbstractions
+{
+    using System;
+    using System.Threading;
+    using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Interfaces;
+
+    public class PlatformThread : IThread
+    {
+        /// <inheritdoc/>
+        public void Run(Action action, PlatformApartmentState apartmentState, bool waitForCompletion)
+        {
+            if (apartmentState == PlatformApartmentState.STA)
+            {
+                throw new ThreadApartmentStateNotSupportedException();
+            }
+
+            if (action == null)
+            {
+                return;
+            }
+
+            Exception exThrown = null;
+            var thread = new Thread(() =>
```

Comment:
If netcore only supports MTA, then no need to create new thread, we can simply invoke action?